src/dnmd/query.c

```diff
@@ -173,6 +173,16 @@ static uint8_t col_to_index(col_index_t col_idx, mdtable_t* table)
     return (uint8_t)idx;
 }
 
+static col_index_t index_to_col(uint8_t idx, mdtable_id_t table_id)
```

Comment:
Was this logic copied from somewhere?

Reply:
No, this is just the reverse of `col_to_index` that's defined above.